tests/fixtures/base-fixtures.ts

```
@@ -0,0 +1,217 @@
+          break;
+        default:
+          logger.setLogLevel(LogLevel.INFO);
+      }
+    } else {
+      logger.setLogLevel(LogLevel.INFO);
+    }
+
+    await use(logger);
+  },
+  // Test metadata with default values
+  testMetadata: async ({}, use: (metadata: BaseTestMetadata) => Promise<void>, testInfo: TestInfo) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)